### PR TITLE
feat: add camera head get request

### PR DESCRIPTION
**Pull request by jhonasiv**
Depends on:
- [ ] https://github.com/tidewise/drivers-deep_trekker/pull/25

### CODE REVIEW

File: tasks/RevolutionTask.cpp [hunk 1]
```diff
@@ -29,6 +29,17 @@ void tryParseAndWriteIgnoringExceptions(function<void()> write_func)
...
+{
+    GetRequestConfig request_config;
+    request_config.update_interval = update_interval;
+    if (!update_interval.isNull()) {
+        request_config.trigger_deadline = Time::now();
+        request_config.request = request;
+    }
+    m_get_requests.push_back(request_config);
+}
+
```
Comment (DeboraTahara) [inline tasks/RevolutionTask.cpp]: If the update_interval is null, do you still have to "push_back" the request_config?

Comment (jhonasiv) [reply tasks/RevolutionTask.cpp]: Actually I dont, that should slightly simplify the sendGetRequests method. Also, just realized that I should clear the m_get_requests in the start of configureHook

**Comment by jhonasiv** [comment]
Ping @afonsofonbraga @seixasxbr @DeboraTahara

File: deep_trekker.orogen [hunk 2]
```diff
@@ -38,6 +38,9 @@ task_context "RevolutionTask" do
     # to loading it from a YAML file), you must transpose it before writing it. See the
     # comments in configureHook for more details.
     property "compensation_matrix", "base/MatrixXd"
+    # Describes what's the update interval to send get requests for specific data in the
+    # API.
+    property "get_requests_intervals", "deep_trekker/GetRequestsIntervals"
```
Comment (seixasxbr) [inline deep_trekker.orogen:43]: In seconds?

**Review by seixasxbr — APPROVED** [review]

File: deep_trekker.orogen (hunk 2 above)
Comment (jhonasiv) [reply deep_trekker.orogen:43]: each interval is a base::Time, so you do it in microseconds

**Pull request merged**